jbpm-workitems/jbpm-workitems-bpmn2/src/test/java/org/jbpm/process/workitem/bpmn2/BusinessRuleTaskTest.java

```diff
@@ -125,6 +125,27 @@ public void testDecisionTaskProcess() throws Exception {
                      processInstance.getState());
     }
 
+    @Test
+    public void testDecisionPassingNullToDMN() throws Exception {
+        //This test make sure that null variables are passed to the DMN execution context, otherwise DMN will throw an exception that an input requirement is missing
+        KieBase kbase = readKnowledgeBase();
+        KieSession ksession = createSession(kbase);
+
+        BusinessRuleTaskHandler handler = new BusinessRuleTaskHandler(GROUP_ID,
+                                                                      ARTIFACT_ID,
+                                                                      VERSION);
+        Map<String, Object> params = new HashMap<String, Object>();
+        params.put("Input",
+                   null);
+
+        WorkflowProcessInstance processInstance = (WorkflowProcessInstance) ksession.startProcess("passthru",
+                                                                                                  params);
+        Object output = processInstance.getVariable("Output");
+
+        assertEquals(null,
+                     output);
+    }
```

Comment:
Just to extend the coverage a little bit, would it be possible to add another test with non-null values? I think this kind of test is not present here yet.

Reply:
I have added a simple test with a non-null value using the same model